themes/kiali/layouts/partials/home/about-section.html

```diff
@@ -1,19 +1,18 @@
   <section id="about">
-    <h1>About Kiali</h1>
+    <h3>About Kiali</h3>
 
     <p>
-    Kiali is a management console for an Istio-based service mesh. It provides
-    dashboards, observability, and lets you operate your mesh with robust
-    configuration and validation capabilities. It shows the structure of your
-    service mesh by inferring traffic topology and displays the health of your
-    mesh. Kiali provides detailed metrics, powerful validation, Grafana access,
-    and strong integration for distributed tracing with Jaeger.
+    Kiali is a management console for Istio service mesh. It provides robust
+    observability for your running mesh, letting you quickly identify issues and
+    then troubleshoot those issues.  Kiali offers in-depth traffic topology, health
+    grades, powerful dashboards, and lets you drill into component detail.
+    Kiali offers correlated views of metrics, logs and tracing, as well as strong
+    validations to pinpoint configuration issues.  Kiali provides several
+    wizards to help you add services to the mesh, define traffic routing, gateways, policies
+    and more.  Kiali includes Grafana and Jaeger integrations.
     </p>
-
-    <p class="buttons">
-    <a class="button secondary" href="https://www.youtube.com/channel/UCcm2NzDN_UCZKk2yYmOpc5w">
-      Check out our demo videos
-    </a>
+    <p>
+    To learn more about Kiali (in under 10 minutes), watch this video...
```

Comment:
I'd suggest to remove the video or just keep only the link.
It's pretty old and it doesn't invite to scroll down the page.